doc/faq.rst

```diff
@@ -313,18 +313,25 @@ being reviewed, you can try:
 
 * follow our :ref:`contribution guidelines <contributing>`, in particular
   :ref:`automated_contributions_policy`, :ref:`filing_bugs`,
-  :ref:`stalled_pull_request` and :ref:`stalled_unclaimed_issues`.
-* complete the provided issue and pull request templates, including a clear and
-  concise description of the issue or motivation for the pull request.
+  :ref:`stalled_pull_request` and :ref:`stalled_unclaimed_issues`,
+* fill in all sections of the issue or pull request template provided by GitHub,
+  including a clear description of the problem or motivation, and any relevant code
+  examples,
 * ensure the title clearly describes the issue or pull request and does not include
   an issue number.
 
 For your pull requests specifically, the following will make it easier to review:
 
-* ensure your PR satisfies all items in the
-  :ref:`Pull request checklist <pr_checklist>`.
-* ensure your PR addresses an issue for which there is clear consensus on the solution.
-* ensure the changes are minimal and directly relevant to the described issue.
+* ensure your PR addresses an issue for which there is clear consensus on the solution
+  (see :ref:`issues_tagged_needs_triage`),
+* ensure the PR satisfies all items in the :ref:`Pull request checklist <pr_checklist>`,
+* ensure the changes are minimal and directly relevant to the described issue,
+* ensure your PR complies with scikit-learn's development conventions (which you can
+  learn about from the docs and by going through similar prior PRs),
```

Comment:
I feel like this is a bit vague and I think the PR checklist already does a reasonable job of discussing some of these conventions: https://scikit-learn.org/dev/developers/contributing.html#pull-request-checklist

Reply:
Ah that makes sense. I can see what you mean, this stuff comes with experience.

I do wonder if these are the main problems with the PRs deserving of this label. e.g., if it's an otherwise well thought out PR, asking them to add param validation would not be a big deal for me. Similarly, I would have thought that PRs that address an issue where a clear path forward has been decided would not result in adding am unwarranted new dependency.